src/sagemaker/jumpstart/curated_hub/accessors/public_hub_s3_accessor.py

```diff
@@ -0,0 +1,121 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module accessors for the SageMaker JumpStart Public Hub."""
+from __future__ import absolute_import
+from sagemaker import model_uris, script_uris
+from sagemaker.jumpstart.curated_hub.utils import (
+    get_model_framework,
+)
+from sagemaker.jumpstart.enums import JumpStartScriptScope
+from sagemaker.jumpstart.types import JumpStartModelSpecs
+from sagemaker.jumpstart.utils import get_jumpstart_content_bucket
+from sagemaker.jumpstart.curated_hub.utils import (
+    get_studio_model_metadata_map_from_region,
+)
+from sagemaker.jumpstart.curated_hub.accessors.s3_object_reference import (
+    S3ObjectReference,
+    create_s3_object_reference_from_bucket_and_key,
+    create_s3_object_reference_from_uri,
+)
+from sagemaker.jumpstart.curated_hub.accessors.jumpstart_s3_accessor import JumpstartS3Accessor
+
+
+class PublicHubS3Accessor(JumpstartS3Accessor):
+    """Helper class to access Public Hub s3 bucket"""
+
+    def __init__(self, region: str):
+        self._region = region
+        self._bucket = get_jumpstart_content_bucket(region)
+        self._studio_metadata_map = get_studio_model_metadata_map_from_region(
+            region
+        )  # Necessary for SDK - Studio metadata drift
```

Comment:
we should fix the sdk metadata in order to avoid this drift, no?

Reply:
Until we have the capability to use just the SDK metadata to get the values we need to access everything, we'll have to keep this studio metadata map. Ideally we wouldn't need it at all